make-util.js

```diff
@@ -129,7 +129,7 @@ var buildNodeTask = function (taskPath, outDir) {
     var originalDir = pwd();
     cd(taskPath);
     if (test('-f', rp('package.json'))) {
-        run('npm install');
+        run('npm install --production');
```

Comment:
yes passed except two tests for VSMobileCenterUpload task. I found they have a dependency wrongly declared under devDependencies, after fixing that they are also passing.